samples/word/01-basics/basic-api-call.yaml

```diff
@@ -13,13 +12,10 @@ script:
         async function run() {
             try {
                 await Word.run(async (context) => {
-                    const range = context.document.getSelection();
+                    let range = context.document.getSelection();
```

Comment:
`const` is better practice in TS.  It's a stronger version of `let`, and has helped me catch real-world bugs before.  https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75